index.html

```diff
@@ -2736,64 +2821,80 @@ <h5><code>subscribeevent</code></h5>
+            id="http-sse-profile-protocol-binding-subscribeevent-6b">
+	    For each message 
             sent, the Web Thing MUST set the <code>event</code> field to the 
             name of the <code>EventAffordance</code> and populate the 
-            <code>data</code> field with event data, if any. The 
+            <code>data</code> field with event data, if any. 
+	  </span>
+	  <span class="rfc2119-assertion" 
+            id="http-sse-profile-protocol-binding-subscribeevent-6c">
+	    The 
             event data MUST follow the data schema specified in the
-            <code>EventAffordance</code> and MUST be serialized in JSON.
+            <code>EventAffordance</code>
+	    and be serialized in JSON.
+	  </span>
+	  <span class="rfc2119-assertion" 
+            id="http-sse-profile-protocol-binding-subscribeevent-6d">
             The <code>id</code> field SHOULD be set to a unique identifier for 
             the event, for use when re-establishing a dropped connection (see 
             below).
-          </p>
+	  </span></p>
           <pre class="example">
             event: overheated\n
             data: 90\n
             id: 12345\n\n
           </pre>
-          <p class="rfc2119-assertion" 
-            id="http-sse-profile-protocol-binding-subscribeevent-7">
+	  <p><span class="rfc2119-assertion" 
+            id="http-sse-profile-protocol-binding-subscribeevent-7a">
             If the connection between the Consumer and Web Thing drops
             (except as a result of the <code>unsubscribe</code> operation 
             defined below), the Consumer MUST re-establish the connection 
             following the steps outlined in the Server-Sent Events specification
-            [[EVENTSOURCE]]. Once the connection is re-established the Web Thing
+            [[EVENTSOURCE]]. 
+	  </span>
+	  <span class="rfc2119-assertion" 
+            id="http-sse-profile-protocol-binding-subscribeevent-7a">
+	    Once the connection is re-established the Web Thing
             SHOULD, if possible, send any missed events which occured since
             the last event specified by the Consumer in a 
             <code>Last-Event-ID</code> header.
-          </p>
+	  </span></p>
         </section>
   
         <section id="http-sse-profile-protocol-binding-events-unsubscribeevent">
           <h5><code>unsubscribeevent</code></h5>
-          <p class="rfc2119-assertion" 
+	  <p><span class="rfc2119-assertion" 
             id="http-sse-profile-protocol-binding-unsubscribeevent-1">
             In order to unsubscribe from an event, a Consumer MUST terminate 
             the corresponding Server-Sent Events connection with the Web Thing 
             as specified in the Server-Sent Events specification
             [[EVENTSOURCE]].
-          </p>
-          <p class="note" title="Terminating a connection using JavaScript">
+	  </span></p>
+	  <p><span class="note" title="Terminating a connection using JavaScript">
```

Comment:
Why the embedded <span>? Seems unnecessary.

Reply:
sure, simplified